src/components/material-buttons/MaterialButton.jsx

```diff
@@ -0,0 +1,34 @@
+import Button from '@mui/material/Button';
+
+export const whiteColor = {
+	color: 'white',
+};
+
+const defaultSx = {
+	fontSize: '1.3rem',
+	...whiteColor,
+};
+
+export const MaterialButton = ({
+	type = 'submit',
+	color,
+	onClick,
+	buttonText,
+	sx = defaultSx,
+	startIcon,
+	variant = 'outlined',
+}) => {
+	return (
+		<Button
+			color={color}
+			fontSize="large"
+			variant={variant}
+			type={type}
+			onClick={onClick}
+			startIcon={startIcon}
+			sx={sx}
+		>
+			{buttonText}
+		</Button>
```

Comment:
Group PR review: curious to know what we're getting out of wrapping these material UI components?

Would it be ok to just import the mui component in each of our components?